File: src/practice/luogu/p1425/Main.kt
// https://www.luogu.com.cn/problem/P1425

fun main() {
  val t = readLine()!!.split(" ").map { it.toInt() }
  val at: Int = (t[2] - t[0] - 1) * 60 + (60 - t[1]) + t[3]
  println("${at / 60} ${at % 60}")
}

File: src/practice/luogu/p1425/Main.py
# https://www.luogu.com.cn/problem/P1425
s = input().split()
a = int(s[0])
b = int(s[1])
c = int(s[2])
d = int(s[3])

h = c - a - 1
m = (60 - b) + d

if m >= 60:
    m -= 60
    h += 1

print(str(h) + ' ' + str(m))
